Changes to Readme

shikhakujur opened the pull request:
Additional lines added to readme.md

shikhakujur committed:
Made changes to the readme.md file

shikhakujur committed:
made some changes

shikhakujur committed:
final change to readme.md

Aniq55 commented:
Pygame comes under Prerequisites I think. So you should mention Pygame too along with Python2

And for running the file, cloning and cd-ing into the cloned directory is enough, I don't think there is a need to create a separate branch for that.

You can add the branch creating steps under the section: #Contributing

After you have made these changes, you can rebase your commits into a single one

Aniq55 commented:
@shikhakujur you didnt have to close it. Just make the changes I've suggested, commit and push.
-_-!

Aniq55 committed:
Corrected syntax

Corrected the syntax for headings, there was no space between the hash and heading

Aniq55 committed:
Fixed broken image link
